lib/active_model/serializer/adapter.rb

```diff
@@ -1,12 +1,14 @@
 module ActiveModel
   class Serializer
     class Adapter
+      UnknownAdapterError = Class.new(ArgumentError)
```

Comment:
Yup, both are common, I'm up for both, but let's keep if in order to keep the way we are doing this across the code